app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListAdapter.kt

```diff
@@ -218,6 +218,15 @@ class ThreadListAdapter @Inject constructor(
 
             draftPrefix.isVisible = hasDrafts
 
+            iconReply.isGone = true
+            iconForward.isGone = true
+
+            if (thread.isAnswered) {
+                iconReply.isVisible = true
+            } else if (thread.isForwarded) {
+                iconForward.isVisible = true
+            }
```

Comment:
Don't assign it twice.
Assign it everytime for the 2 values, to the correct value.